docs/howto/configure/pangeo-requester-pays-access.md

```diff
@@ -0,0 +1,103 @@
+# Pangeo Data Access via Requester Pays
+
+For some hubs, such as our Pangeo deployments, the communities they serve require access to data stored in other projects.
+Accessing data normally comes with a charge that the folks _hosting_ the data have to take care of.
+However, there is a method by which those making the request are responsible for the charges instead: [Requester Pays](https://cloud.google.com/storage/docs/requester-pays).
+This document shows the steps required to set this method up.
+
+## Setting up Requester Pays Access on GCP
+
+```{note}
+We may automate these steps in the future.
```

Comment:
This definitely feels like something that is semi-common for research groups, and that is not scalable in its current implementation of manual steps :-)

Reply:
Absolutely. We should open an issue to figure out how to automate this as I suspect that it can't be confined to just the terraform scripts or just the deployer module, as for the last step you need a hub up and running. In practice though, this is a 5 min manual job.